openfeature/version.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.7.0"
```

Comment:
What is this for?

Reply:
this gives the possibility to import the SDK version on runtime and add it to the user agent. nothing crucial for the release itself.

Reply:
Should we have Release Please manage this?

Reply:
Yeah, I think it should automatically do that, at least that's how I understood their documentation.

Reply:
Okay, sounds good. Let's give it a shot.